pthread_stop_world.c

```diff
@@ -434,24 +434,6 @@ GC_INNER void GC_push_all_stacks(void)
   int GC_stopping_pid = 0;
 #endif
 
-#ifdef PLATFORM_ANDROID
```

Comment:
I don't think we will be going back to an older NDK, and we won't back port this change. So I would rather not check.